### DISCUSSION TITLE
is there anyway to move my current window activities to specialworkspace ?

### DISCUSSION THREAD

**Discussion by Greatz08** [hyprwm/Hyprland#8864] [General]
Like i could use mod + shift + workspace_number to move the current workspace activities to that workspace but if i wanna do same can i do that with special workspaces too ? if yes then how if no then will we ever get feature to move current workspace activities to specialworkspace ? 

Like in my scenario i always want to move my one firefox window instance to that special workspace but unfortunately i dont know if there is a way to do so. i dont want to close that window always and then from scratch recreate same window in specialworkspace manually

**Comment by izmyname** [comment]
```
# Special workspace (scratchpad)
bind = $mainMod, S, togglespecialworkspace, magic
bind = $extraMod, S, movetoworkspace, special:magic
```
